src/main.js

```diff
@@ -11,6 +11,46 @@ exports.loadFile = function(path) {
     return exports.load(readFileSync(path, 'utf-8'))
 }
 
+/*
+Priority
+- Definition without infrequent keyword
+- Surname
+- Modern Variant
+- Old Variant
+*/
+
+const sortEntries = (entries) => {
+    return [...entries].sort((a, b) => {
+        const aIsOldVariant = a.english.includes("old variant of ");
```

Comment:
Can you adhere to the style of the existing code?

- No semicolons
- Use `let` instead of `const` inside functions